CMakeLists.txt

```diff
@@ -79,40 +73,15 @@ if(ENABLE_TESTING)
     enable_testing()
     add_subdirectory("test")
 else()
-    file(GLOB BM_COMMON_MESSAGES_SOURCES
-        bm_common_messages/*.c
-        bm_common_messages/*.cpp
-    )
     set(SOURCES
-        src/bcmp.c
-        src/config.c
-        src/device.c
-        src/heartbeat.c
-        src/info.c
-        src/neighbors.c
         src/l2.c
-        src/middleware.c
-        src/packet.c
-        src/ping.c
-        src/pubsub.c
-        src/resource_discovery.c
-        src/time.c
-        src/topology.c
-        src/cbor_service_helper.c
-        src/dfu/dfu_client.c
-        src/dfu/dfu_core.c
-        src/dfu/dfu_host.c
-        src/echo_service.c
-        src/sys_info_service.c
-        src/config_cbor_map_service.c
-        src/bm_service.c
-        src/bm_service_request.c
         ${CMAKE_CURRENT_LIST_DIR}/third_party/tinycbor/src/cborparser.c
         ${CMAKE_CURRENT_LIST_DIR}/third_party/tinycbor/src/cborencoder_float.c
         ${CMAKE_CURRENT_LIST_DIR}/third_party/tinycbor/src/cborencoder.c
         ${CMAKE_CURRENT_LIST_DIR}/third_party/tinycbor/src/cborerrorstrings.c
         ${CMAKE_CURRENT_LIST_DIR}/third_party/tinycbor/src/cborvalidation.c
-        ${BM_COMMON_MESSAGES_SOURCES}
+        ${CMAKE_CURRENT_LIST_DIR}/third_party/crc/crc16.c
+        ${CMAKE_CURRENT_LIST_DIR}/third_party/crc/crc32.c
```

Comment:
Do we want to create a 3rd party library?

Reply:
Per slack discussion, I'd vote no. The main value of building static libraries is to prove to ourselves that we've minimized the dependencies in our own code. We already know these 3rd party libraries don't depend on bm_core. 😄